### PR TITLE
feat: added italian regions country map

### PR DESCRIPTION
**Pull request by romanfed**

### CODE REVIEW

File: plugins/legacy-plugin-chart-country-map/src/countries.ts [hunk 1]
```diff
@@ -79,6 +81,9 @@ export const countryOptions = Object.keys(countries).map(x => {
   if (x === 'uk' || x === 'usa') {
     return [x, x.toUpperCase()];
   }
+  if (x === 'italy_regions') {
+    return [x, 'Italy (regions)'];
```
Comment (villebro) [inline plugins/legacy-plugin-chart-country-map/src/countries.ts:85]: I believe this should be translated:
```suggestion
    return [x, t('Italy (regions)')];
```

**Review by villebro — CHANGES_REQUESTED** [review]

File: plugins/legacy-plugin-chart-country-map/src/countries.ts (hunk 1 above)
Comment (romanfed) [reply plugins/legacy-plugin-chart-country-map/src/countries.ts:85]: If I'm not wrong, there aren't translations for country map names. I think adding translations should be another task.

**Comment by junlincc** [comment]
how can we get this PR through? @villebro

File: plugins/legacy-plugin-chart-country-map/src/countries.ts (hunk 1 above)
Comment (villebro) [reply plugins/legacy-plugin-chart-country-map/src/countries.ts:85]: Fair enough - let's do a translation sweep later when that becomes a priority.

**Review by villebro — APPROVED** [review]

**Comment by ktmud** [comment]
What's the difference between "Italy" and "Italy (regions)"? Can we add more PR description just for context?

**Pull request merged**